### PR TITLE
Mehtap/css

### PR DESCRIPTION
**Pull request by mehtapy**

### CODE REVIEW

File: client/src/components/pages/faq/FAQ.tsx [hunk 1]
```diff
@@ -125,9 +127,35 @@ const FAQ = () => {
...
+      const msg = new SpeechSynthesisUtterance(selectedText);
+      window.speechSynthesis.speak(msg); // Seçilen metni sesli olarak oku
+    }
+  }, [selectedText, isTTSActive]);
+
+  const handleTTSButtonClick = () => {
+    setTTSActive(!isTTSActive);
+  };
+
+  const toggleTTS = () => {
```
Comment (gitadventurer241) [inline client/src/components/pages/faq/FAQ.tsx:143]: 1. why this file is being modified?
2. why is being modified in this PR?
3. from which design the textTopSpeech part is this following?
4. why there are comments not in english?

File: client/src/components/layout/authenticated/Authenticated.tsx [hunk 2]
```diff
@@ -67,8 +68,9 @@ const Authenticated = ({ content }: { content: JSX.Element }) => {
     paddingTop: 64,
     paddingRight: 40,
     paddingLeft: 40,
-    overflow: "initial",
+    //overflow: "initial",
     transition: "margin-left 0.3s",
```
Comment (gitadventurer241) [inline client/src/components/layout/authenticated/Authenticated.tsx:72]: why did you leave commented code?

File: client/src/components/UI/filter/FilterSelect.tsx [hunk 3]
```diff
@@ -55,9 +55,13 @@ const FilterSelect: React.FC<multiOptionProps> = ({
...
   };
 
+  const customSelectStyle = {
+    width: '100%',
+    minWidth:"140px"
+  };
   return (
     <Select
-      style={{ width: 200 }}
+      style={customSelectStyle}
```
Comment (gitadventurer241) [inline client/src/components/UI/filter/FilterSelect.tsx:64]: could you simply add the `minWidth` to `style={{ width:"100%", minWidth: "140px" }}`?

**Review by gitadventurer241 — CHANGES_REQUESTED** [review]

File: client/src/components/layout/authenticated/Authenticated.tsx (hunk 2 above)
Comment (mehtapy) [reply client/src/components/layout/authenticated/Authenticated.tsx:72]: deleted

File: client/src/components/UI/filter/FilterSelect.tsx (hunk 3 above)
Comment (mehtapy) [reply client/src/components/UI/filter/FilterSelect.tsx:64]: done

**Pull request closed without merging**